packages/main/src/pages/static/contact-us/components/ContactFormContainer.jsx

```diff
@@ -31,14 +31,13 @@ function ContactFormContainer() {
     success: ""
   });
 
-  useEffect(() => {
-    getUserInfo().then(userInfo => {
-      setUserAuth(userInfo?.user.email ? userInfo.user : {});
-      setValues(values => ({
-        ...values,
-        email: userAuth.email ? userAuth.email : values.email
-      }));
-    });
+  useEffect(async () => {

```

Comment:
Your work was to add contact link to the bottom of the page. How is this code related to the work you were to do?

Reply:
Hi, please I am a bit confused as the question isn’t clear. A mentor told me I should make the contact us page below the page and above functional. Please can you shed more light on the question so I can make the corrections. Thanks

Reply:
I've confirmed your task. Yours was to remove the link to contact from the navigation bar. It should only be at the bottom of the page @christofa

Reply:
Hi Sau, I have made all the updates you requested